mne/commands/mne_show_fiff.py

```diff
@@ -21,11 +21,6 @@ def run():
     if len(args) != 1:
         parser.print_help()
         sys.exit(1)
-    # This works around an annoying bug on Windows for show_fiff, see:
-    # https://pythonhosted.org/kitchen/unicode-frustrations.html
-    if int(sys.version[0]) < 3:
-        UTF8Writer = codecs.getwriter('utf8')
-        sys.stdout = UTF8Writer(sys.stdout)
```

Comment:
I don't know if this fixes the readability, but this is the only way I'm able to use this command with the problematic file. All the other configurations seem to fail.